draft-kuehlewind-taps-crypto-sep.txt

```diff
@@ -0,0 +1,672 @@
+
```

Comment:
Did you mean to commit the .txt?

Reply:
Yeah, that was intentional. I can remove it though if you don't want it tracked.